src/Microsoft.AspNetCore.Antiforgery/AntiforgeryTokenSet.cs

```diff
@@ -23,22 +23,13 @@ public class AntiforgeryTokenSet
             string formFieldName,
             string headerName)
         {
-            if (string.IsNullOrEmpty(requestToken))
-            {
-                throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, nameof(requestToken));
-            }
```

Comment:
I changed `DefaultAntiforgery.Serialize` to prove that it will never return a null token.